scripts/create_dataset.py

```diff
@@ -1,69 +1,161 @@
-                                        sub_directory="assets")
-    save_dataset_as_json(dataset_with_assets,
-                         output_name)
-
-
-def init_dataset(dataset_name, dataset_description):
-    """
-    Creates any empty dataset as a dict that has a study name, description, and schema.
-    :param dataset_name: the schema name for the dataset
-    :param dataset_description: description of the schema
-    """
+    input_tables = get_json(parent_directory=input_tables_directory)
+    tables, relationships = create_tables_and_relationships(input_tables=input_tables)
+
+    input_assets = get_json(parent_directory=input_assets_directory)
+    assets = [
+        {
+            "name": input_assets[input_asset_name]["name"],
+            "rootTable": input_assets[input_asset_name]["rootTable"],
+            "rootColumn": input_assets[input_asset_name]["rootColumn"],
+            "tables": [{"name": table["name"], "columns": []} for table in tables],
+            "follow": [relationship["name"] for relationship in relationships]
+        } for input_asset_name in input_assets
+    ]
+
     dataset = {
         "name": dataset_name,
         "description": dataset_description,
-        "schema": {
-            "tables": [],
-            "relationships": [],
-            "assets": []
-        }
+        "schema": {"tables": tables, "relationships": relationships, "assets": assets}
     }
-    return dataset
+    save_dataset_as_json(dataset=dataset, output_name=output_name)
 
 
-def add_to_schema(dataset, schema_type, parent_directory,  sub_directory):
+def get_json(parent_directory):
     """
-    Adds JSON files to the current dataset as a dict.
-    :param dataset: the combined dataset to be written to disk
-    :param schema_type: which part of the data set is being added to the schema, (either assets, relationships or tables)
-    :param parent_directory: the directory containing assets, relationships and tables JSON files
-    :param sub_directory: the child directory of either assets or tables JSON files
+    Adds JSON files to a dict with each key being the name field of a json file.
+    and the value being the corresponding json.
+    :param parent_directory: the directory containing JSON files to add to the dict
     """
-    json_files_path = os.path.join(parent_directory, sub_directory)
-    for file in os.listdir(json_files_path):
+    input_dict = {}
+    for file in os.listdir(parent_directory):
         if file.endswith(".json"):
-            with open(os.path.join(json_files_path, file), "r") as json_file:
+            with open(os.path.join(parent_directory, file), "r") as json_file:
                 json_as_dict = json.load(json_file)
-            dataset["schema"][schema_type].append(json_as_dict)
-    return dataset
+            # add the name field field of the json as a kay to make each json searchable
+            input_dict.update({json_as_dict["name"]: json_as_dict})
```

Comment:
Simpler could be:
```suggestion
            input_dict[json_as_dict["name"]] = json_as_dict
```
Update is a little heavy for just one element